utils/tracking.ts

```diff
@@ -0,0 +1,73 @@
+import { RetrievedService } from "@pagopa/io-functions-commons/dist/src/models/service";
+import { initTelemetryClient } from "./appinsight";
+
+/**
+ * Track when an incoming service document is invalid
+ *
+ * @param telemetryClient
+ * @returns
+ */
+export const trackIgnoredInvalidIncomingDocument = (
+  telemetryClient: ReturnType<typeof initTelemetryClient>
+) => (
+  d: unknown = {} /** default empty object to prevent nullish values */,
+  reason: string = ""
+): void => {
+  telemetryClient.trackEvent({
+    name: "selfcare.subsmigrations.services.invalid-incoming-document",
+    properties: {
+      documentId: (d as RetrievedService).id,
+      message: "Invalid document received",
```

Comment:
Since the message is static, and it can be derived from the event name, do we need it?